packages/cursorless-engine/src/generateSpokenForm/defaultSpokenForms/marks.ts

```diff
@@ -0,0 +1,64 @@
+import { PartialMark } from "@cursorless/common";
+
+const hatColors: Record<string, string | null> = {
```

Comment:
switched this back to `string`, to keep this PR more self-contained.  We can revisit decoration spoken form generation in follow-up